src/Xabe.FFmpeg/Conversion/Conversion.cs

```diff
@@ -675,5 +675,20 @@ public IConversion SetPixelFormat(string pixelFormat)
                 _outputPixelFormat = $"-pix_fmt {pixelFormat} ";
             return this;
         }
+
+        /// <inheritdoc />
+        public IConversion VisualiseAudio(VideoSize size, 
```

Comment:
Move this method to Conversion/Snippets/Audio.cs as internal and expose it through Snippets.
There are only few cases to use that and I think it would be nice to help user create visualisation, but not in main conversion builder.
In my opinion, this method should take audio input, parameters from this method and output file. It will return IConversion which create new file with 2 streams - original audio stream and visualised video stream.

Reply:
I think I have done it correctly. Could you please take a look.